SimpleAkkaDotNetCluster/SimpleCluster/akka-hocon.conf

```diff
@@ -1,53 +1,18 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
-<configuration>
-
-  <configSections>
-    <section name="akka"
-             type="Akka.Configuration.Hocon.AkkaConfigurationSection, Akka" />
-  </configSections>
-
-
-   <akka>
-    <hocon>
-      <![CDATA[
-
-        akka {
-            extensions = ["Akka.Cluster.Tools.Client.ClusterClientReceptionistExtensionProvider, Akka.Cluster.Tools"]
-            actor {
-              provider = cluster
-
-                serializers {
-                    akka-cluster-client = "Akka.Cluster.Tools.Client.Serialization.ClusterClientMessageSerializer, Akka.Cluster.Tools"
-                  }
-                  serialization-bindings {
-                    "Akka.Cluster.Tools.Client.IClusterClientMessage, Akka.Cluster.Tools" = akka-cluster-client
-                  }
-                  serialization-identifiers {
-                    "Akka.Cluster.Tools.Client.Serialization.ClusterClientMessageSerializer, Akka.Cluster.Tools" = 15
-                  }
-
-            }
+﻿akka {
```

Comment:
A HOCON configuration have a JSON-like syntax, there's no need to embed it inside an XML, the XML syntax is a deprecated legacy way of embedding HOCON inside the old .NET Framework configuration file.

Reply:
This HOCON settings have been trimmed to the bare minimum required to run this project.